.buildkite/test-pipeline.yaml

```
@@ -455,6 +456,7 @@ steps:
   - vllm/
   - tests/models/language
   commands:
+    - pip install mteb
```

```suggestion
    - pip install mteb<2
```
`MTEB` is preparing `v2` update that would break thests